GOVERNANCE.md

```diff
@@ -181,24 +181,54 @@ yourself from the MAINTAINERS file.
 ## Removal of inactive maintainers
 
 Similar to the procedure for adding new maintainers, existing maintainers can
-be removed from the list if they do not show significant activity on the
-project. Periodically, the maintainers review the list of maintainers and their
-activity over the last year.
-
-If a maintainer has shown insufficient activity over this period, a neutral
-person will contact the maintainer to ask if they want to continue being
-a maintainer. If the maintainer decides to step down as a maintainer, they
-open a pull request to be removed from the MAINTAINERS file.
-
-If a committer is unable to continue participating in project level votes but
-wishes to remain a maintainer on the project, they may convert their role to
-a reviewer until they are able to perform committer duties again.
-
-If an inactive maintainer wants to remain in their current role or does not open
-their own pull request to change roles, another maintainer may open up a pull
-request to the MAINTAINERS file to change their role. Role changes and removals
-must be approved by 2/3 of the current committers. The voting and discussion
-period is seven days, after which a committer will verify the votes,
+be removed from the list if they do not show consistent activity on the
+project. Depending on the existing role, committer or reviewer, there are
+variations to the removal process detailed as follows.
+
+### Committer removal
+
+For committers, the maintainers periodically review committer project
+activity. The maintainers use the CNCF "devstats" community metrics tool for
+data on user activity statistics.
+
+If a committer has shown no activity over a six-month period, the committer
+will be contacted to notify them of their proposed Emeritus role. There is
+no automatic change of status in the project. First, the activity status is
+discussed directly with the committer, and second, maintainers discuss whether
+other non-tracked contributions to the project reflect an ongoing, active
+participation in the project.
+
+Based on the outcome of these discussions, the committer may choose to remove
+themselves from the MAINTAINERS file, or another maintainer may open a
+pull request to the MAINTAINERS file on their behalf for removal.
+Removals must be approved by 2/3 of the current committers. The voting
+and discussion period is seven days, after which a committer will verify the votes,
+merge the pull request, and apply the changes to the organization.
+
+### Emeritus maintainers
+
+For committers who are stepping down, the project would like to memorialize their
+contributions to the project by recognizing them as Emeritus maintainers in
+the EMERITUS.md file. The EMERITUS.md file will include a brief paragraph
+summarizing their contribution to the containerd project and recognize them as
+permanent Emeritus members of the community.
+
+If in the future an Emeritus maintainer has the desire or ability to return to
```

Comment:
I wonder if this needs to require any demonstration of new activity in the project?

Reply:
I think an emeritus committer volunteering to become more active is sufficient; if they're unable to follow-through with that we can handle that through discussion.

Reply:
agree with Samuel, need to make this as easy/painless as possible